benchexec/tablegenerator/react-table/src/components/LinkOverlay.js

```diff
@@ -28,11 +28,10 @@ export default class LinkOverlay extends React.Component {
   //    (this fails for ZIPs on the local disk).
   // 3) Try loading url from within ZIP archive without Range header.
   loadContent = async url => {
-    console.log("load content", url);
```

Comment:
The logging of this method has been quite useful in various circumstances, please do not remove it.